### ISSUE TITLE
Module Request: `word`

### ISSUE THREAD

**Issue by Marak** [Marak/faker.js#1096]
We'd like to see another top module added for `word`.

This could include the following methods to start:

```
noun
pronoun
verb
adjective
adverb
preposition
conjunction
interjection
```

We don't have to include the entire dictionary, but it would be good to have at around 1,000 nouns and 1,000 verbs to start. Will want a good amount of each other type of word as well.

This would be a great Pull Request for someone looking to contribute to the project.

**Comment by zerefwayne** [comment]
@Marak This sounds like something I needed for my project last year. I would be glad to work on adding this one!

**Comment by Marak** [comment]
Closed with #1172

Thanks @ad-walker !
